sdks/typescript/package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "apache-beam",
-      "version": "2.54.0-SNAPSHOT",
+      "version": "2.57.0-SNAPSHOT",
```

Comment:
Strange, there is still this change, which might cause the failure of typescript check?

Reply:
I think I finally understood what happened here: there's two lines with the package version, and I failed to notice the second one.